selfnotify/notifier.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 import urllib
+import urllib.parse
```

Comment:
This should be compatible with `python2` too (not this will work only for `python3`)